ecpy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -56,6 +57,27 @@ def clear_sequence(self):
         with self.secure():
             self._AWG.write('SOURCE{}:WAVEFORM ""'.format(self._channel))
 
+    @secure_communication()
+    def set_sequence_pos(self, name, position):
+        """Sets the sequence index position to waveform name.
+
+        """
+        self._AWG.clear_output_buffer()
+        try:
+            current_length = int(self._AWG.ask("SEQuence:LENGth?"))
+        except:
```

Comment:
NEVER use bare exception as it catches SystemError and KeyboardInterrupt which is not a good idea. To catch general exceptions use ```except Exception:```

Reply:
ok